projects/plugins/boost/app/assets/src/js/features/page-cache/meta/meta.tsx

```diff
@@ -38,25 +36,28 @@ const Meta = () => {
 		}
 
 		if ( totalBypassPatterns === 0 && ! logging ) {
-			return __( 'No exceptions or logging.', 'jetpack-boost' );
+			return __( 'No exceptions.', 'jetpack-boost' ) + ' ' + __( 'No logging.', 'jetpack-boost' );
+		}
+
+		let loggingMessage;
+		if ( logging ) {
+			loggingMessage = __( 'Logging activated.', 'jetpack-boost' );
+		}
+
+		if ( ! logging ) {
+			loggingMessage = __( 'No logging.', 'jetpack-boost' );
 		}
 
 		return (
-			<>
-				{ totalBypassPatterns > 0 ? (
-					<>
-						{ sprintf(
-							/* translators: %d is the number of cache bypass patterns. */
-							_n( '%d exception.', '%d exceptions.', totalBypassPatterns, 'jetpack-boost' ),
-							totalBypassPatterns
-						) }
-					</>
-				) : (
-					__( 'No exceptions.', 'jetpack-boost' )
-				) }{ ' ' }
-				{ logging && __( 'Logging activated.', 'jetpack-boost' ) }
-				{ ! logging && __( 'No logging.', 'jetpack-boost' ) }
-			</>
+			( totalBypassPatterns > 0
+				? sprintf(
+						/* translators: %d is the number of cache bypass patterns. */
+						_n( '%d exception.', '%d exceptions.', totalBypassPatterns, 'jetpack-boost' ),
+						totalBypassPatterns
+				  )
+				: __( 'No exceptions.', 'jetpack-boost' ) ) +
```

Comment:
That closing bracket not aligning with the opening one is bugging me.

```suggestion
				: __( 'No exceptions.', 'jetpack-boost' )
			) +
```